exosphere-shared/src/docker-helper.ls

```diff
@@ -29,7 +29,8 @@ class DockerHelper
 
 
   @get-docker-ip = (container) ->
-    child_process.exec-sync("docker inspect --format '{{ .NetworkSettings.IPAddress }}' #{container}", "utf8") if DockerHelper.container-exists container
+    if DockerHelper.container-exists container
+      child_process.exec-sync("docker inspect --format '{{ .NetworkSettings.IPAddress }}' #{container}", "utf8") |> (.to-string!) |> (.replace '\n', '') 
```

Comment:
Better to call [trim](https://www.w3schools.com/jsref/jsref_trim_string.asp) on the string rather than removing line breaks manually. On windows they look differently.